pyccel/codegen/printing/pycode.py

```diff
@@ -1136,6 +1135,12 @@ def _print_TypingFinal(self, expr):
         annotation = self._print(expr.arg)
         return f'const {annotation}'
 
+    def _print_ListPop(self, expr):
+        args = ','.join(self._print(a) for a in expr.args[1:])
+        name = self._print(expr.args[0])
+        method = self._print(expr.name)
+        return f"{name}.{method}({args})"
```

Comment:
Do you foresee a case where this would be something other than:
```suggestion
        return f"{name}.pop({args})"
```
?
For example an object which inherits from `pop`? If not then I don't think that there is anything to be gained from using `self._print` which incurs a cost